src/libraries/Common/tests/AndroidTestRunner/AndroidTestRunner.cs

```diff
@@ -92,7 +92,9 @@ protected override IEnumerable<TestAssemblyInfo> GetTestAssemblies()
 
     public string? Locale { get; }
 
+#pragma warning disable CS8764
```

Comment:
This is due to: https://github.com/dotnet/xharness/pull/198 -- once there is a xharness build with that change we can remove this.

Reply:
I'll fix this in a separate PR to update xharness.